### PR TITLE
filtering nfts on queries

### PR DESCRIPTION
**Pull request by vgvishesh**

### CODE REVIEW

File: src/modules/nft/components/search-nft/open-api.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+					},
+					owner: "97a3giHcGsk8YoEgWv4rP1ooWwJBgS72fpckZM6mQiFH",
+				},
+			],
+		},
+	},
+};
+
+export function SearchAttributesOpenApi() {
+	return applyDecorators(ApiOperation({ summary: 'Read all NFT' }), ApiOkResponse(SearchAttributesResponse));
```
Comment (codinpsycho) [inline src/modules/nft/components/search-nft/open-api.ts:57]: Summary needs to be updated.

File: src/modules/nft/components/search-nft/search-nft.service.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { NftInfoAccessor } from "src/dal/nft-repo/nft-info.accessor";
+import { getNftDbResponseFromNftInfo } from "src/dal/nft-repo/nft-info.helper";
+
+@Injectable()
+export class SearchNftService {
+	constructor(private nftInfoAccessor: NftInfoAccessor) { }
+	async searchNftsByAttributes(query: any, apiKeyId: ObjectId): Promise<any> {
+		const filter = {}
+		for (let key in query) {
+			console.log(query[key]);
```
Comment (codinpsycho) [inline src/modules/nft/components/search-nft/search-nft.service.ts:12]: Can remove console logs?

File: src/modules/nft/components/search-nft/open-api.ts (hunk 1 above)
Comment (vgvishesh) [reply src/modules/nft/components/search-nft/open-api.ts]: done

File: src/modules/nft/components/search-nft/search-nft.service.ts (hunk 2 above)
Comment (vgvishesh) [reply src/modules/nft/components/search-nft/search-nft.service.ts]: done

**Pull request merged**